### ISSUE TITLE
Get ready for CRAN release

### ISSUE THREAD

**Issue by statnmap** [ThinkR-open/testdown#10]
Follow: https://github.com/ThinkR-open/prepare-for-cran

**Comment by csgillespie** [comment]
Just wondering if you are still planning a CRAN release? Alternatively, could you make tagged release :)

Thanks!

**Comment by statnmap** [comment]
I do not think we will release for CRAN, but we definitely can realize a tagged release that you can install from r-universe.  
Let me try to deal with these Windows/MacOS issues on the CI.

**Comment by statnmap** [comment]
Done. 
You can now use tag V0.1.2: https://github.com/ThinkR-open/testdown/releases/tag/v0.1.2
and install from our r-universe:
```r
install.packages("testdown", 
                 repos = c("thinkropen" = "https://thinkr-open.r-universe.dev"))
```

**Comment by csgillespie** [comment]
Thanks! Much appreciated.

**Comment by rpodcast** [comment]
My group just discovered this great package and it fulfills a big need for us! Just checking if there's any possibility it could make its way to CRAN? Certainly the tagged release and availability on R-Universe is a huge help, but in my industry it's often much easier for us to install from CRAN over alternative repositories.

**Comment by DianeBeldame** [comment]
I'll let @statnmap answer about planning a CRAN release (focus is rather on `{fusen}` at the moment 😩). In the meantime, I'm super curious about what kind of needs it fulfills : would you be so kind as to enlighten me @rpodcast 😇? (and maybe @csgillespie)

**Comment by kellijohnson-NOAA** [comment]
Our team at NOAA Fisheries uses this package to report the testing of multiple R packages. I am not so concerned with a new release on CRAN but I am curious about the maintainership of this package and if it will continue to be developed?
